src/zenml/zen_server/routers/workspaces_endpoints.py

```diff
@@ -1064,7 +1028,7 @@ def create_code_repository(
 @handle_exceptions
 def get_workspace_statistics(
     workspace_name_or_id: Union[str, UUID],
-    _: AuthContext = Security(authorize, scopes=[PermissionType.READ]),
+    _: AuthContext = Security(authorize),
```

Comment:
More of a consistency issue: the stats should reflect the number of items shown in the various lists (stacks, components, pipelines, runs) for the current user. There's no filtering done here however, so the number of items will be >= than those visible to the active user.

Reply:
Fixed, we can now pass filter models to the `count_...()` methods of the `SQLZenStore` which allows us to apply RBAC (and also any other) filters.

Reply:
Love it ! This could even become one of the CRUD functions that we implement for every entity, so we'll have `count` in addition to create/read/list/update/delete. CRLCUD !!

Reply:
Haha CRLCUD 😅